build/actions.sh

```diff
@@ -12,6 +12,7 @@ set -o errexit
 source build/common.sh
 
 write-release-date() {
+  mkdir -p _build
```

Comment:
Yes good point, I added a comment about this:

https://github.com/oilshell/oil/commit/74c0356e162337afa73a80f7fa4735de31d7a126

I'm trying to be a little principled about how I use the `Makefile` and `build/{compile,actions}.sh`. That is, try to use them in a purely functional way, but Make doesn't really work like that so it could be a false hope.  Ideally the build system would make directories for you and you wouldn't have to worry about it.